README.md

```diff
@@ -1,11 +1,15 @@
-# Spacemesh CLI Wallet
+# Spacemesh CLIWallet
 
 ## Overview
-A basic reference Spacemesh CLI Wallet designed to work together with a [go-spacemesh full node](https://github.com/spacemeshos/go-spacemesh).
+A basic reference console Spacemesh Wallet designed to work together with a [go-spacemesh full node](https://github.com/spacemeshos/go-spacemesh).
 
-The CLI Wallet is designed for developers who want to hack on the Spacemesh platform. For non-devs we recommend using Smapp - the Spacemesh App. [Smapp](https://github.com/spacemeshos/smapp) is available for all major desktop platforms and includes a wallet.
+The CLIWallet is designed for developers who want to hack on the Spacemesh platform. For non-devs we recommend using Smapp - the Spacemesh App. [Smapp](https://github.com/spacemeshos/smapp) is available for all major desktop platforms and includes a wallet.
 
 ## Functionality
+The wallet is a Spacemesh node api 2.0 client and implements basic wallet features. You can create a new coin account, execute transactions, check account balance and view transactions./
+
+> WARNING: CLIWallet is currently insecure as it saves private keys in cleartext on your local store. It is not yet a production-quality wallet. We plan to update the project to store private data securely in future releases.
```

Comment:
Add a link here to an issue on this topic?

Reply:
done